proto/forwarding/forwarding_table.proto

```diff
@@ -175,6 +181,19 @@ message BridgeTableDesc {
   uint32 transient_timeout = 1;
 }
 
+message ActionTableDesc {
+}
+
+message ActionEntryDesc {
+  enum Type {
+    TYPE_UNSPECIFIED = 0;
+    TYPE_PREPEND = 1;
+    TYPE_APPEND = 2;
+  }
+  string id = 1;
+  Type insert_type = 2;
+}
+
```

Comment:
where is the data to be inserted? Should this be called InsertionEntryDesc? Action is the most generic term.

Reply:
its ActionEtntryDesc because the table is call ActionTable (its a list of actions). this follow pattern of other tables